app/posts/personalised-prevention-platform/2025/03/2025-03-31-technical-research.md

```diff
@@ -0,0 +1,120 @@
+- Personal Demographics Service (PDS)
+- General Practice Extraction Service (GPES)
+- Hospital Episodes Statistics (HES)
+- Systemic Anti-Cancer Therapy (SACT)
+- Radiotherapy Treatment Data Set (RTDS)
+
+#### What NHS England technology we can use to communicate with cohorts of people
+
+The conclusion is that we can use the NHS Notify platform to communicate with cohorts of people for national campaigns. This is because:
+
+NHS Notify is a national level strategic platform for messaging and communicating with citizens via SMS messages, emails, NHS App messages, and even physical letters.
+
+It can easily integrate with CaaS and therefore use it to communicate with cohorts of people created by the latter.
+
+It can also easily integrate with the NHS App, to send messages directly to its users. These messages can appear in the NHS App inbox and are accompanied by push notifications, such as banner or badge alerts, to ensure recipients are aware of new communications.
+
+## What we learned
+
+### We can already create people cohorts with shared characteristics and risk factors
+
+The following would still need to be approved by the NHS Information Governance and Clinical teams.
+
+There is a healthy level of maturity in the technology that allows us to define cohorts of people with shared characteristics and risk factors. CaaS, combined with a range of tools and technologies already used for immunisation campaigns, could be reused to create campaigns with a focus on personalised prevention.
+
+The key considerations are:
+
+1.  NHS CaaS and NHS Notify are live, and already supporting the rollout of campaigns for prevention (for example Immunisations).
+2. NHS Data sets and collections are ready and available through CaaS, but subject to NHS Information Governance, Legal and Clinical engagement and approval.
+
+## What we are doing next
+
+### If we want to create people cohorts with shared characteristics and risk factors
+
+The next steps are well defined given the high level of maturity of CaaS and its pre‑existing integrations to other national platforms (for example NHS Notify).
+
+1. The following is needed to create new citizen or patient cohorts for prevention campaigns:
+2. Refine the scope of the proposed cohorts and the data requirements.
+3. Formally engage all relevant NHS platforms teams (including CaaS).
+4. Secure and make funding available for all the required activities.
+5. Agree the legal basis for accessing and using the required data.
+6. Complete Information Governance (IG) activities resulting in an approved Data Protection Impact Assessment (DPIA).
+7. Complete clinical assurance of the proposed cohorts.
+8. Create, test and assure the proposed cohorts.
+
+## What opportunities does this offer?
+
+Important – Subject to NHS Information Governance and Clinical approval, the technologies and data researched can enable opportunities to create cohorts of people with shared characteristics and risk factors.
```

Comment:
I would delete the "Important – " bit...?

Reply:
👍🏻